Header header
float32[] data # 3x1 array of [temperature, heartbeat, respiration]
#float32 temperature
#float32 heartbeat
#float32 respiration
